src/query/functions-v2/tests/it/scalars/testdata/string.txt

```diff
@@ -3138,3 +3138,957 @@ evaluation (internal):
 +--------+--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
 
 
+ast            : regexp_instr('dog cat dog', 'dog', 1)
```

Comment:
can we move `regexp_xxx` functions to a new test file?

Reply:
ok